transpiler/src/main/java/org/jsweet/transpiler/TypeScript2JavaScriptWithTscTranspiler.java

```diff
@@ -18,14 +18,16 @@
 
     private Process tsCompilationProcess;
 
+    //Note that hangingTscTimeout is ignored
     @Override
     protected synchronized void doTranspile( //
             ErrorCountTranspilationHandler transpilationHandler, //
             Collection<File> tsFiles, //
             Collection<SourceFile> tsSourceFiles, //
             JSweetOptions options, //
             boolean ignoreErrors, //
-            OnTsTranspilationCompletedCallback onTsTranspilationCompleted) throws Exception {
+            OnTsTranspilationCompletedCallback onTsTranspilationCompleted,//
+            int hangingTscTimeout) throws Exception {
```

Comment:
no need to pass this param since its getter is defined in options, which is already a param of this method